examples/tutorials/tracing/llm/trace-llm.ipynb

```diff
@@ -133,7 +133,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Run the LLM application again, and you should be able to see new trace logged in the trace UI, and it is clickable to see more details."
+    "Run the LLM application again, and you should be able to see new trace logged in the trace UI, and it is clickable to see more details.\n",
+    "\n",
+    "![trace-detail](./media/trace-detail.png)"
```

Comment:
add an extra folder so it's less likey to conflict with other samples.
./media/trace-llm/trace-detail.png

Reply:
updated following this.